app/jobs/dqt/recommend_for_award_job.rb

```diff
@@ -8,22 +8,16 @@ class RecommendForAwardJob < ApplicationJob
     def perform(trainee)
       return unless FeatureService.enabled?(:integrate_with_dqt)
 
-      if trainee.hesa_record?
-        message = "Trainee id: #{trainee.id}, slug: #{trainee.slug} has been recommended for award but is a HESA trainee"
-        username = "Register Trainee Teachers: Job Failure"
-        SlackNotifierService.call(message:, username:)
-      else
-        award_date = RecommendForAward.call(trainee:)
+      award_date = RecommendForAward.call(trainee:)
 
-        if award_date
-          trainee.award_qts!(award_date)
-          Trainees::Update.call(trainee: trainee, update_dqt: false)
-        else
-          raise(
-            DqtNoAwardDateError,
-            "failed to retrieve award date from DQT for trainee: #{trainee.id}",
-          )
-        end
+      if award_date
+        trainee.award_qts!(award_date)
+        Trainees::Update.call(trainee: trainee, update_dqt: false)
```

Comment:
does something need to change with `update_dqt: false` ? I think we still won't send to DQT